tests/unit/test_algebraic_expression.cpp

```diff
@@ -40,6 +41,8 @@ class AlgebraicExpressionTest: public ::testing::Test {
         assert(GrB_init(GrB_NONBLOCKING) == GrB_SUCCESS);
         srand(time(NULL));
 
+        // Use the malloc family for allocations
+        Alloc_Reset();
```

Comment:
For `make test` can we undefine REDIS_MODULE_TARGET ?

Also I'm missing the definition of REDIS_MODULE_TARGET in our /Makefile, the only place which I've see
this flag being defined is in deps/GraphBLAS/CMakeLists.txt
Am I missing something?